.eslintrc.js

```diff
@@ -1,10 +1,34 @@
 module.exports = {
+  root: true,
   extends: '@mate-academy/eslint-config',
+  plugins: ['react', 'jest'],
   env: {
-    jest: true
+    es2021: true,
   },
   rules: {
-    'no-proto': 0
+    'no-proto': 0,
```

Comment:
The rule 'no-proto' is set to 0, which means it is turned off. Ensure this is intentional, as using '__proto__' can lead to security vulnerabilities and unexpected behavior.